db/seeds.rb

```diff
@@ -21,4 +21,12 @@
     password: password,
     password_confirmation: password)
 
+#Following relationships
+users = User.all
+user = users.first
+following = users[2..50]
+follower = users[3..40]
```

Comment:
Please change the spelling. This should be "followers"
![image](https://user-images.githubusercontent.com/35232348/95971779-3b46e900-0e44-11eb-8a92-a80ffd718293.png)